pkg/daemon/file_writers.go

```diff
@@ -276,10 +276,11 @@ func lookupGID(group string) (int, error) {
 // This is essentially ResolveNodeUidAndGid() from Ignition; XXX should dedupe
 func getFileOwnership(file ign3types.File) (int, int, error) {
 	uid, gid := 0, 0 // default to root
+	var err error    // create default error var
```

Comment:
minor nit: I think the comment can be skipped in this case :smile: but no harm in having it

Reply:
I almost didn't put it there :P originally had it set to nil with a comment explaining why but the verify test didn't like that since that is the default value, so I removed it. should've removed the whole comment in hindsight